lib/SIL/IR/OperandOwnership.cpp

```diff
@@ -216,14 +220,14 @@ OPERAND_OWNERSHIP(UnownedInstantaneousUse, UnmanagedAutoreleaseValue)
 OPERAND_OWNERSHIP(PointerEscape, ProjectBox) // The result is a T*.
 OPERAND_OWNERSHIP(PointerEscape, ProjectExistentialBox)
 OPERAND_OWNERSHIP(PointerEscape, UncheckedOwnershipConversion)
+OPERAND_OWNERSHIP(PointerEscape, ConvertEscapeToNoEscape)
```

Comment:
The cvt escape to no escape change makes sense.

Reply:
Right, the general comment elsewhere that PointerEscape operands are basically bugs in the design that need to be fixed applies equally to convert_escape_to_no_escape. Obviously the original closure should have a well-defined scope.